indico/modules/rb_new/client/js/modules/bookRoom/BookRoom.jsx

```diff
@@ -436,6 +469,7 @@ const mapStateToProps = (state) => {
         isInitializing: globalSelectors.isInitializing(state),
         queryString: stateToQueryString(state.bookRoom, qsFilterRules, qsBookRoomRules),
         showMap: globalSelectors.isMapVisible(state),
+        dateRange: state.bookRoom.timeline.data.dateRange
```

Comment:
I believe there is already a `getTimelineDateRange` selector for this in `bookRoom/selectors.js`.